### PR TITLE
Adapt example script for LTI certification to code changes

### PR DESCRIPTION
**Pull request by marcospri**
For:

- https://github.com/hypothesis/product-backlog/issues/1515

This is not a very satisfying approach but the whole certification process is a bit wonky.

---
Our grading service has changed leaving the example script out of date.

Modify the script to be up to date with the current codebase.

### CODE REVIEW

File: tests/functional/lti_certification/v13/grading/test_assignment_and_grading.py [hunk 1]
```diff
@@ -32,13 +32,17 @@ def test_it(self):
         ```
         # These are included in the JWT sent by the testing tool
         # https://purl.imsglobal.org/spec/lti/claim/resource_link -> id
-        RESOURCE_LINK_ID = "7a3f7bed2844419caa4edf74c93f4375"
+        RESOURCE_LINK_ID = "d57dfc1975a441da896c9d7eb0413b96"
```
Comment (marcospri) [inline tests/functional/lti_certification/v13/grading/test_assignment_and_grading.py:35]: Updated in re-certification

File: tests/functional/lti_certification/v13/grading/test_assignment_and_grading.py [hunk 2]
```diff
@@ -48,15 +52,18 @@ def test_it(self):
         )
         ltia_service = LTIAHTTPService(
             application_instance.lti_registration,
+            MiscPlugin(),
             request.find_service(JWTService),
             request.find_service(name="http"),
+            request.find_service(JWTOAuth2TokenService),
```
Comment (marcospri) [inline tests/functional/lti_certification/v13/grading/test_assignment_and_grading.py:58]: New dependencies here

File: tests/functional/lti_certification/v13/grading/test_assignment_and_grading.py (hunk 1 above)
Comment (marcospri) [inline tests/functional/lti_certification/v13/grading/test_assignment_and_grading.py:35]: Note that this whole script is a comment on this file.

**Review by jon-betts — COMMENTED** [review]
I'm not really sure how to review this to be honest?

One of my suggestions was going to be to put it into it's own file next to this test with no-cover on it, but I realise it doesn't actually work as a script as it expects to `db` and `request` to exist. So I think this is to be used with `make shell`?

I was going to suggest putting it into it's own script as:

 * It would mean we could spot errors more easily
 * Might be easier to run
 
 I suspect it wouldn't be too hard to convert this into something you could run with one of the `pyramid` helpers, but maybe it's not worth it?

**Pull request merged**